sparkle/src/_index.ts

```diff
@@ -148,6 +148,9 @@ export { Markdown };
 import { DataTable } from "./components/DataTable";
 export { DataTable };
 
+import { Table } from "./components/Table";
```

Comment:
`nit` I'd go with DataTable since it's more complex than a regular "table".

Reply:
We already have a `DataTable` component